Client/Client.csproj

```diff
@@ -36,17 +43,26 @@
     <ProjectReference Include="..\Core\Core.csproj" />
   </ItemGroup>
 
+  <Target Name="SetPublishDir" BeforeTargets="Build">
+    <PropertyGroup>
+      <PublishDir Condition="'$(SelfContainedRelease)' == 'true'">$(HelionRootDir)\Publish\$(RuntimeIdentifier)_SelfContained\</PublishDir>
+      <PublishDir Condition="'$(SelfContainedRelease)' != 'true'">$(HelionRootDir)\Publish\$(RuntimeIdentifier)\</PublishDir>
+    </PropertyGroup>
+  </Target>
+
   <Target Name="GetSupportFiles">
     <ItemGroup>
       <SoundFonts Include="$(ProjectDir)SoundFonts\**\*.sf2" />
 
       <!-- Runtime specified, e.g. "-r win-x64"; copy native files into root -->
       <NativeFiles Include="$(ProjectDir)Unmanaged\win-x64\**\*" Destination="%(RecursiveDir)" Condition="'$(RuntimeIdentifier)' == 'win-x64'" />
```

Comment:
This is all just stuff to make sure the libfluidsynth native file gets copied when building for Linux.